feature/gribi/ate_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -160,12 +164,29 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) *ondatra.ATETopology {
 	return top
 }
 
+// Configure Network instance
+func configNetworkInstance(t *testing.T, dut *ondatra.DUTDevice, vrfname string, intfname ...string) {
+	d := &oc.Root{}
+	ni := d.GetOrCreateNetworkInstance(vrfname)
+	ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	if len(intfname) != 0 {
```

Comment:
Your func() API claims it multiple interface names, so your func() should either do a `for loop` to support it, or gate on `if len(intfName) == 1` and error out if it's not. I think a `for loop` is better. (otherwise why you want your func API supports the list of interface? :) )

Reply:
intfname is used as an optional argument (variadic args)  in function configNetworkInstance , such that same method can used to configure vrf with and without interface config

Reply:
My point is that your func API is not consistent is with what it does, e.g. if it's a list why you simply just do `intfname[0]`?

Reply:
updated